### PR TITLE
feat: add client demo

### PR DESCRIPTION
**Pull request by gabrielmatau79**
introduce cliente Based pymediasoup library. 
Resolve issue #5

### CODE REVIEW

File: demo/pymediasoup-client/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    ffmpeg\
+    && rm -rf /var/lib/apt/lists/*
+
+# Set the working directory in the container
+WORKDIR /app
+
+# Copy the current directory contents into the container at /app
+COPY /src /app
+
+COPY /patch /patch
```
Comment (genaris) [inline demo/pymediasoup-client/Dockerfile:18]: Now that pymediasoup 1.0.1 has been released, we need to remove the patch (and pypatch from requirements)

File: demo/pymediasoup-client/patch/pymediasoup.patch
Comment (genaris) [inline demo/pymediasoup-client/patch/pymediasoup.patch:1]: Don't forget to remove the patch since it is not needed anymore

**Review by genaris — CHANGES_REQUESTED** [review]
Once we remove the usage of pypatch (as pymediasoup 1.0.1 has been released today and therefore gives us an error when trying to apply the aptch), the demo works great. However, there are a few things on the behaviour that should be updated to mimick UnicID Issuer DTS flow needs:

- the HTTP POST request to join-call should return immediately (i.e. it should not block until the video is finished). So as soon as it validates the input parameters, it should run the `Demo` task asynchronously and return an HTTP 200. Then, it will be the Demo task the one responsible for calling success_url if video is played properly or failure_url if there is a problem
- success_url and failure_url are optional. If not defined, it simply won't call anything

File: demo/pymediasoup-client/README.md [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+## Usage
+
+`/join-call` Endpoint
+
+- URL: /join-call
+- Method: POST
+- Description: Connects to a Mediasoup WebRTC server and plays the video specified by DEFAULT_VIDEO_SRC_URL.
+- Parameters:
+    - ws_url: The WebSocket URL to connect to the Mediasoup server. (Required)
+    - success_url: URL to notify when the video finishes successfully. (Required)
```
[the remark this answers is not in the document]
Comment (gabrielmatau79) [reply demo/pymediasoup-client/README.md:51]: Solved!  success_url and failure_url now are optional!

File: demo/pymediasoup-client/Dockerfile (hunk 1 above)
Comment (gabrielmatau79) [reply demo/pymediasoup-client/Dockerfile:18]: Solved! Remove patch into Dockerfile

File: demo/pymediasoup-client/patch/pymediasoup.patch
Comment (gabrielmatau79) [reply demo/pymediasoup-client/patch/pymediasoup.patch:1]: Removed the pymediasoup patch as well as the pypatch package.

**Review by genaris — APPROVED** [review]

**Pull request merged**